packages/integrations/image/components/Picture.astro

```diff
@@ -1,16 +1,22 @@
 ---
 import { getPicture } from '../dist/index.js';
-import type { ImageAttributes, ImageMetadata, OutputFormat, PictureAttributes, TransformOptions } from '../dist/types';
+import type { ImageMetadata, OutputFormat, TransformOptions } from '../dist/index.js';
 
-export interface LocalImageProps extends Omit<PictureAttributes, 'src' | 'width' | 'height'>, Omit<TransformOptions, 'src'>, Pick<ImageAttributes, 'loading' | 'decoding'> {
+interface LocalImageProps extends
```

Comment:
This was a nitpick I had during the bug bash, if Astro attributes are not excluded (`set:html` and co), they'll show up twice in the completions of the component

Granted, the JSX definitions could be better here (HTML attributes shouldn't include Astro stuff), so maybe it's a me thing to fix, really